### PR TITLE
Update import wallet route

### PR DESCRIPTION
**Pull request by IViol**
JWWEB-97

- `create wallet` fixes (remove recompose/pure, update flow types, etc)
- update `import wallet` (module, saga, view)
- move base functionality for create/import routes into `wallets` module/saga

### CODE REVIEW

File: src/routes/Wallets/routes/Create/sagas/walletsCreate.js [hunk 1]
```diff
@@ -6,117 +6,96 @@ import { put, select, takeEvery } from 'redux-saga/effects'
...
+    testPasswordData,
+  }: WalletsState = walletsData
 
-  const isPasswordExists: boolean = !!walletsData.testPasswordData
+  const isPasswordExists: boolean = !!testPasswordData
 
   if (!isPasswordExists) {
+    if (password === name) {
+      yield put(
+        wallets.setInvalidField('password', 'Password should not be equal with wallet name'),
```
Comment (flexxnn) [inline src/routes/Wallets/routes/Create/sagas/walletsCreate.js]: optional: to be moved to i18n

File: src/routes/Wallets/routes/Create/sagas/walletsCreate.js [hunk 2]
```diff
@@ -6,117 +6,96 @@ import { put, select, takeEvery } from 'redux-saga/effects'
...
     passwordConfirm,
-  }: WalletsCreateState = yield select(selectWalletsCreate)
+    testPasswordData,
+  }: WalletsState = walletsData
 
-  const isPasswordExists: boolean = !!walletsData.testPasswordData
+  const isPasswordExists: boolean = !!testPasswordData
 
   if (!isPasswordExists) {
+    if (password === name) {
```
Comment (flexxnn) [inline src/routes/Wallets/routes/Create/sagas/walletsCreate.js]: optional: use name.trim() to remove unused spaces

File: src/routes/Wallets/routes/Create/sagas/walletsCreate.js (hunk 1 above)
Comment (IViol) [reply src/routes/Wallets/routes/Create/sagas/walletsCreate.js]: we have JWWEB-18 for that

File: src/routes/Wallets/routes/Create/sagas/walletsCreate.js (hunk 2 above)
Comment (IViol) [reply src/routes/Wallets/routes/Create/sagas/walletsCreate.js]: we have JWWEB-159 for that

Comment (Romanior) [reply src/routes/Wallets/routes/Create/sagas/walletsCreate.js]: well if its one line, better to fix it now, instead of creating a new ticket.

Comment (IViol) [reply src/routes/Wallets/routes/Create/sagas/walletsCreate.js]: The issue has already created. And we must to discuss little bit more with designer & PO about behaviour of this field

File: src/routes/Wallets/routes/Create/sagas/walletsCreate.js [hunk 3]
```diff
@@ -152,34 +127,15 @@ export function* setPrevStep(): Saga<void> {
...
-export function* watchCreateWalletCreateSuccess(): Saga<void> {
-  yield takeEvery(CREATE_SUCCESS, createSuccess)
+function* clean(): Saga<void> {
+  yield put(wallets.clean())
 }
 
-export function* watchCreateWalletCreateError(): Saga<void> {
-  yield takeEvery(CREATE_ERROR, createError)
+export function* walletsCreateRootSaga(): Saga<void> {
+  yield takeEvery(walletsCreate.CLEAN, clean)
```
Comment (Romanior) [inline src/routes/Wallets/routes/Create/sagas/walletsCreate.js]: cool, thanks!!

**Pull request merged**